src/test/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalControllerTest.java

```diff
@@ -275,7 +276,7 @@ public void test_RetrievePaymentAccountByEmailFromHeader() {
         verify(httpRequest, times(3)).getHeader(anyString());
     }
 
-    @Test(expected = EmptyResultDataAccessException.class)
+    @Test(expected = ResourceNotFoundException.class)
```

Comment:
if Junit 5 is being used, we could probably use `assertThrows` instead of the above.

Reply:
We haven't migrated to junit 5 since we use serenity reporting which doesn't support junit5.